frontend/package.json

```diff
@@ -58,7 +58,7 @@
     "start-js": "react-scripts start",
     "build": "INLINE_RUNTIME_CHUNK=false REACT_APP_CURRENT_COMMIT=$(git rev-parse HEAD) SASS_PATH=$(cd ./node_modules && pwd):$(cd ./node_modules/@uswds && pwd):$(cd ./node_modules/@uswds/uswds/packages && pwd):$(cd ./src/scss && pwd) react-scripts build",
     "postbuild": "echo 'pushstate: enabled' > build/Staticfile && echo $(git rev-parse HEAD) > build/static/commit.txt && cd build && mv *touchpoints*.* static/",
-    "codegen": "graphql-codegen --config codegen.yml && prettier --write 'src/generated/graphql.tsx'",
+    "codegen": "graphql-codegen --config codegen.yml && prettier --write 'src/generated/graphql.tsx' && eslint --fix 'src/generated/graphql.tsx' ",
```

Comment:
fyi for folks: this codegen command was missing the `eslint` part for the longest time and was causing linting issues in the diff that we check between the generated files. Added this here so that everything works in one command